Fix CSS on onboarding page

samrae7 opened the pull request:
- remove height from onboarding-main
- remove height from onboarding-content

<!--- Prepend PR title with [WIP] if work in progress. Remove when ready for review. -->

<!--- For a timely review/response, please avoid force-pushing additional commits if your PR already received reviews or comments -->

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [X] Bug Fix
- [ ] Documentation Update

## Description

Fix CSS bug on onboarding page. See issues #4485

## Related Tickets & Documents

Fixes #4485

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

There are 9 variants for this page. I have checked all of them in Chrome desktop and mobile view.

Removing `height` from `.onboarding-content` fixes the issue with the button overlapping in mobile view. This created an issue with the desktop view ( button also overlapping) which is resolved by also removing `height` from `.onboarding-main`

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

**Screenshots of the fix on my local branch**
<img width="1083" alt="Screenshot 2019-10-23 at 08 22 11" src="https://user-images.githubusercontent.com/4126146/67459759-34223c00-f631-11e9-9ad2-c1c2f9ac270e.png">
<img width="1131" alt="Screenshot 2019-10-23 at 08 22 30" src="https://user-images.githubusercontent.com/4126146/67459792-43a18500-f631-11e9-8a40-01dbd33e611c.png">

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/thepracticaldev/dev.to?pullRequest=4512) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/thepracticaldev/dev.to?pullRequest=4512) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/thepracticaldev/dev.to?pullRequest=4512) it.</sub>

samrae7 commented:
Sure i'll do it tonight. Thanks

On Wed, 30 Oct 2019, 21:27 Jacob Herrington, <notifications@github.com>
wrote:

> *@jacobherrington* requested changes on this pull request.
>
> I agree with @maestromac <https://github.com/maestromac>! Can we wrap
> this in a media query?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/thepracticaldev/dev.to/pull/4512?email_source=notifications&email_token=AA7PLQQJFTUYIXAMA3EP73TQRH35JA5CNFSM4JCZARAKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCJZRUHY#pullrequestreview-309533215>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA7PLQXBWDJPTQKMO34AON3QRH35JANCNFSM4JCZARAA>
> .
>

samrae7 commented:
@jacobherrington - I added the media query. Tested on chrome and the css fix is still in place for mobile but `.onboarding-main` height is now fixed (I no longer need to remove the fixed height on `.onboarding-main`  because we are still setting height on `.onboarding-content when screen size is > 800px)

samrae7 commented:
> Thank you for this PR @samrae7 !
> 
> While this does solve the issue, it takes away the fixed steady window for content of each onboarding page. It wasn't obvious to me immediately until I start comparing it with master. Perhaps there should only be fixed height when the screen is at certain size.

@maestromac sorry I thought I had replied to your comment before saying I'd make that change but I see now that I didn't. Apologies

samrae7 commented:
Screenshots with latest change:

<img width="419" alt="Screenshot 2019-11-01 at 09 47 56" src="https://user-images.githubusercontent.com/4126146/68017335-67e30e80-fc8e-11e9-95cc-c22072e2046d.png">
<img width="1284" alt="Screenshot 2019-11-01 at 09 48 41" src="https://user-images.githubusercontent.com/4126146/68017357-70d3e000-fc8e-11e9-8c6f-45bb64455da1.png">
